hack/test.sh

```diff
@@ -29,3 +29,12 @@ kubectl delete namespace argocd-e2e
 kubectl create namespace argocd-e2e-cluster-config
 kubectl kuttl test --config kuttl-test-cluster-config.yaml
 kubectl delete namespace argocd-e2e-cluster-config
+
+# if target cluster is ocp, run kuttl-test-rhsso.yaml
+# else run kuttl-test-keycloak.yaml
+isOCP=$(kubectl get crds | grep "openshiftapiservers.operator.openshift.io" || true)
+if [[ ! -z ${isOCP} ]]; then
+    kubectl kuttl test --config kuttl-test-rhsso.yaml
+else
+    kubectl kuttl test --config kuttl-test-keycloak.yaml
+fi
```

Comment:
We are running them in GitHub Actions CI right?  Can you please take the openshift tests to downstream e2e tests automation?

Reply:
Sure, I have plans to port them to downstream as well. I would like to keep the tests here as well, downstream has almost zero RHSSO code, so every time we need a change to RHSSO, we will do it here in the upstream. 

Agree, GitHub actions CI would never run the openshift e2e tests as it would not have the OpenShift API(openshiftapiservers.operator.openshift.io). I will remove the `isOCP` condition from here and let the CI run only k8s tests.